engine/schema/src/com/cloud/vm/dao/UserVmDaoImpl.java

```diff
@@ -192,6 +192,18 @@ void init() {
                 JoinBuilder.JoinType.INNER);
         AccountDataCenterVirtualSearch.done();
 
+        SearchBuilder<NicVO> nicSearchByNetwork = _nicDao.createSearchBuilder();
```

Comment:
@rhtyd: Perfect! Thanks for finding out. I will submit a new pull request with the proper fix.